data/results/batch_integration_embed/immune_batch.json

```diff
@@ -0,0 +1,639 @@
+            "Batch ASW": 0.5997084099207267, 
+            "Isolated label Silhouette": 1.9664934538152996, 
+            "kBET": NaN
+        }, 
+        {
+            "Name": "FastMNN embed (full/unscaled)", 
+            "Paper": "A description of the theory behind the fastMNN algorithm", 
+            "Paper URL": "https://marionilab.github.io/FurtherMNN2018/theory/description.html", 
+            "Year": 2019, 
+            "Library": "https://doi.org/doi:10.18129/B9.bioc.batchelor", 
+            "Implementation": "https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/fastmnn.py", 
+            "Version": "1.12.3", 
+            "Runtime (min)": 4.8, 
+            "CPU (%)": 130.5, 
+            "Memory (GB)": 7.1, 
+            "Rank": 5, 
+            "Cell Cycle Score": 1.2162968264690746, 
+            "kBET": NaN, 
+            "PC Regression": 0.8114072636355143, 
+            "Isolated label Silhouette": 4.105145074766431, 
+            "Batch ASW": 0.5165209542029467, 
+            "Silhouette": 0.9756664452278917
+        }, 
+        {
+            "Name": "Combat (full/unscaled)", 
+            "Paper": "Adjusting batch effects in microarray expression data using empirical Bayes methods", 
+            "Paper URL": "https://academic.oup.com/biostatistics/article/8/1/118/252073", 
+            "Year": 2007, 
+            "Library": "https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html", 
+            "Implementation": "https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/combat.py", 
+            "Version": "1.9.1", 
+            "Runtime (min)": 4.316666666666666, 
+            "CPU (%)": 1307.0, 
+            "Memory (GB)": 17.6, 
+            "Rank": 6, 
+            "Cell Cycle Score": 0.9054627478956384, 
+            "Batch ASW": 0.7080889340451098, 
+            "PC Regression": 0.8890849209219234, 
+            "Isolated label Silhouette": 1.0774622312580522, 
+            "kBET": NaN, 
+            "Silhouette": 1.1644768959612157
+        }, 
+        {
+            "Name": "Combat (full/scaled)", 
+            "Paper": "Adjusting batch effects in microarray expression data using empirical Bayes methods", 
+            "Paper URL": "https://academic.oup.com/biostatistics/article/8/1/118/252073", 
+            "Year": 2007, 
+            "Library": "https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html", 
+            "Implementation": "https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/combat.py", 
+            "Version": "1.9.1", 
+            "Runtime (min)": 5.966666666666667, 
+            "CPU (%)": 1223.9, 
+            "Memory (GB)": 23.6, 
+            "Rank": 7, 
+            "Cell Cycle Score": 1.1485731501654854, 
+            "kBET": NaN, 
```

Comment:
@LuckyMD @danielStrobl these NaNs are causing deploy errors. Thoughts on what we should do about them?

Reply:
Okay... PCR nan's I don't know why they're there yet... kBET nan's I think I found out. If `library(kBET)` doesn't work, then `np.nan` is returned. This is something we didn't notice initially, and I just started an issue (https://github.com/theislab/scib/issues/349) on it. It seems that kBET is not added as a requirement to `openproblems-r-extras`, but it didn't complain here as it just returned `np.nan` instead.

Reply:
Do we have any install_github functionality for r installs? kBET would just need to be added via:
`install_github('theislab/kBET')`

We could also just remove the metric for now as it wasn't properly added